src/main/java/frc/team2412/robot/commands/arm/ArmDiagnosticCommand.java

```diff
@@ -0,0 +1,59 @@
+package frc.team2412.robot.commands.arm;
+
+import static frc.team2412.robot.commands.arm.SetWristCommand.WristPosition.*;
+import static frc.team2412.robot.subsystems.ArmSubsystem.ArmConstants.PositionType.*;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.team2412.robot.subsystems.ArmSubsystem;
+
+public class ArmDiagnosticCommand extends SequentialCommandGroup {
+	public ArmDiagnosticCommand(ArmSubsystem armSubsystem) {
+		addCommands(
+				new SetArmCommand(armSubsystem, LOW),
+				new WaitCommand(0.5),
+				new SetWristCommand(armSubsystem, WRIST_RETRACT_CUBE),
+				new WaitCommand(0.5),
+				new SetWristCommand(armSubsystem, WRIST_RETRACT_CONE),
+				new WaitCommand(0.5),
+				new SetWristCommand(armSubsystem, WRIST_SCORE),
+				new WaitCommand(0.5),
+				new SetWristCommand(armSubsystem, WRIST_SCORE),
```

Comment:
Is there a reason why `new SetWristCommand(armSubsystem, WRIST_SCORE)` is done twice here (and throughout the rest of the diagnostic command)?